docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,333 @@
+Datasets
+==========
+
+Revisiting the Boston Housing Dataset
+-------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The Boston Housing dataset is one of three datasets currently callable in 
+fairlearn's *datasets* module. In the past, it has commonly been used for 
+benchmarking in popular machine learning libraries, including scikit-learn and 
+open-ml. However, as the machine learning community has developed awareness 
+about fairness issues with popular benchmarking datasets, the Boston Housing 
+data has been phased out of many libraries. We migrated the dataset to  
+fairlearn after it was phased out of scikit-learn in June 2020. The dataset  
+remains in fairlearn as an example of how system racism can occur in data  
+and to show the effect of fairlearn's assessment and pre-/in-/post-processing  
+tools on real, problematic data. 
+
+**We wrote this blog post to achieve the following goals:**
+  * Educate users about the history of the dataset and its bias
+  * Show users how bias manifests in the data and in downstream modelling tasks
+  * Suggest best practices for dealing with the Boston Housing data and 
+  alternative benchmarking datasets
+
+.. _dataset_origin:
+
+Dataset Origin and Use
+^^^^^^^^^^^^^^^^^^^^^
+
+Contrary to expectation, the Boston Housing dataset was not developed for 
+economic purposes. `Harrison and Rubenfield (1978) <https://deepblue.lib.umich.edu/bitstream/handle/2027.42/22636/0000186.pdf?sequence=1&isAllowed=y>` 
+developed the dataset to illustrate the issues with using housing market data 
+to measure consumer willingness to pay for clean air. The authors use a 
+`hedonic pricing <https://www.investopedia.com/terms/h/hedonicpricing.asp>` 
+approach, which assumes that the price of a good or service can be modeled as a 
+function of features both internal and external to the good or service. The 
+input to this model was a dataset comprising the Boston Standard Metropolitan 
+Statistical Area, with the *NOX* variable serving as a proxy for air quality. 
```

Comment:
This is the first mention of NOX, so perhaps let's just spell it out as nitric oxide concentration with NOX in parentheses.

Reply:
Related: Adding words means that now all the lines in the same paragraph need to be adjusted to stay below 80 characters. That's super annoying. I would know because I did that hundreds of times. @riedgar-ms recommended starting a new line for every sentence because of this, so that you save yourself that effort. Restructured text only uses new lines in the rendered text if you leave a blank line, so it won't be visible. Anyway, just a suggestion!

Reply:
I spelled out NOX. Thanks for the line-editing tip! I am new to the restructured text format so I appreciate any tips.